content/post/091-dev-exp.md

```diff
@@ -0,0 +1,36 @@
+
+* Improve how users and developers get help, reducing response times and improving access to existing information and documentation.
+* Reduce the number of pull requests and issue reports that stay in limbo by giving consistent feedback on status, prioritization and team capacity.
+* Better inform contributors about the different steps of the contribution process, and how they fit with the priorities of the project and core working groups.
+Increase community participation, providing better avenues to facilitate involvement in the projects, remove bottlenecks, and create capacity to give support.
+
+After discussion with active IPFS contributors and research into other OSS community best practices, we’ve identified a few changes to help improve the IPFS contributor experience:
+
+* Contributors need a clear and explicit contribution process to the more than 100 active repositories in the IPFS organization. Any new contribution should have clarity on who is going to review the submission, what priority it has, and what the next steps are. To meet this need, we’ll be drafting a new process in the [official IPFS forums](http://discuss.ipfs.io/) so that it can be iterated on by the community.
+* The new IPFS Github organization labelling taxonomy (recently rolled out to all repositories) aims to improve triaging of discussions and requests to clearly reflect status and priority. This comes with an additional effort to consolidate and clean up outdated issues and inactive repositories. As a result, many inactive repos have been moved to the [ipfs-inactive Github organization](https://github.com/ipfs-inactive) to clearly differentiate code that is actively maintained from code that is dormant or deprecated.
+* To help funnel users and devs where the community can better support them, our [IPFS discussion forums](https://discuss.ipfs.io/) will be more prominently displayed (in docs, READMEs and issue templates) as the default venue for support (along with pointers to [ipfs.io/help](https://ipfs.io/help)). Some discussion-only and meta-topic repositories are also in the process of migrating their conversations to the forums, where they can reach a wider audience and be more easily searchable and discoverable by other users.
+* New contributors to repositories will now receive automated responses on their first submission, giving context on the process outlined above and reminders for needed information. This will help reduce the number of round trips for contributors and reviewers/maintainers.
+* A [major review of the documentation](https://docs-beta.ipfs.io/) site is already underway to provide better developer documentation for our major implementations and products. [Watch this space!](https://github.com/ipfs/docs/milestones)
+
```

Comment:
do you want to add an image here of the labelling taxonomy, response bot, or similar?